gitman/models/config.py

```diff
@@ -339,6 +341,11 @@ def load_config(start=None, *, search=True):
     if start:
         start = os.path.abspath(start)
     else:
+        if os.name == 'nt':
+            is_subst, realpath = _detect_windows_subst_drive()
+            if is_subst:
+                os.chdir(realpath)
+
         start = os.getcwd()
```

Comment:
Would it be safe to call `os.chdir(os.realpath(start))` on all platforms?

Reply:
For windows and linux it is safe so I expect that macos is ok as well - but I don't know for sure.

But os.realpath(start) would just return 'R:\\' in python below version 3.8.
Before version 3.8 symbolic links and junctions was not resolved on Windows.
Details here: [https://docs.python.org/3/library/os.path.html?highlight=realpath#os.path.realpath](https://docs.python.org/3/library/os.path.html?highlight=realpath#os.path.realpath)

That's why the solution is a bit ugly.

Are willing to change python requirement to 3.8+?

Reply:
Yeah, it would require a major version bump, but I was planning to do that anyway before https://github.com/jacebrowning/gitman/pull/248 gets released.

Reply:
Cool I will rework the solution then.